tests/cephadm/test_cephadm_pull.py

```diff
@@ -0,0 +1,70 @@
+import ast
+
+import requests
+
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def getting_ceph_version(link):
+    """
+    Method return ceph version from magna002 latest ceph container yaml
+    Args:
+        link: magna002 latest ceph container yaml link
+    Returns:
+        ceph version
+    """
+
+    response = requests.get(link)
+    ceph_version = " "
+    for data in (response.text).split("\n"):
+        if "ceph-version" in data:
+            ceph_version = data
+            break
+    return ceph_version
+
+
+def run(ceph_cluster, **kw):
+    """
+    Args: Check the latest image version using cephadm pull option
+        **kw: Key/value pairs of configuration information to be used in the test.
+    Returns:
+        0 - if test case pass
+        1 - it test case fails
+    Test case covered : CEPH-83573787
+    Test Case Flow:
+    1. Deploy a 5.x cluster
+    2. check for container image latest available using cephadm pull option.
+    """
+
+    try:
+        config = kw.get("config")
+
+        log.info("Executing cephadm pull command")
+        instance = CephAdmin(cluster=ceph_cluster, **config)
+
+        # Geting ceph version detail using cephadm pull command
+        out, _ = instance.installer.exec_command(cmd="cephadm pull", sudo=True)
+        ceph_version_cephadm = ast.literal_eval(out)["ceph_version"].split(" ")[2][:-6]
+        log.info("cepadm pulled ceph version %s image" % ceph_version_cephadm)
+
+        # Geting ceph version detail from magna002 latest ceph container yaml
+        ceph_version = getting_ceph_version(
+            "http://magna002.ceph.redhat.com/cephci-jenkins/latest-rhceph-container-info/RHCEPH-5.1.yaml"
+        ).split(" ")[-1]
+        log.info(
+            "Ceph version %s image available in ceph container yaml" % ceph_version
+        )
+
+        # Verfication cephadm pulling latest ceph version image
+        if ceph_version == ceph_version_cephadm:
+            log.info("cephadm pull verification succesfully")
+            return 0
+        log.info("cephadm pull verification unsuccesfully")
```

Comment:
Provide more detailed info for the failure.

Reply:
Thanks @sunilangadi2 
Updated more log message

Reply:
@MohitBis can you use line breaker and cover all info in one `log.info`?

Reply:
Thanks @sunilangadi2
Updated